xmipp

```diff
@@ -634,6 +642,38 @@ def downloadDeepLearningModels(cwd, dedicatedMode=False):
         ok = pDLdownload.poll() is None or pDLdownload.poll() == 0
     return ok
 
+
+def scipionConnection(dirname):
+    scipionSoftware = os.environ.get('SCIPION_SOFTWARE',
+                                     os.path.join(getScipionHome(), 'software'))
+    scipionLibs = os.path.join(scipionSoftware, 'lib')
+    scipionBindings = os.path.join(scipionSoftware, 'bindings')
+    scipionSoftwareEM = os.path.join(scipionSoftware, 'em')
+    xmippHomeLink = os.path.join(scipionSoftwareEM, 'xmipp')
```

Comment:
this should lead to `xmipp/build`, not just `xmipp`

Reply:
I think "xmipp" here is the link name. I think is correct

Reply:
dirname lead to build or the installation folder the user pass as arg.